### ISSUE TITLE
command-line option format for utility scripts

### ISSUE THREAD

**Issue by macieksmuga** [ga4gh/server#203]
the python utility scripts such as `process_schemas.py` have some inconsistency in their command-line options, using POSIX style dash separators for some, and camel casing for others:
```
$ python scripts/process_schemas.py -h
usage: process_schemas.py [-h] [--outputFile OUTPUTFILE]
                          [--avro-tools-jar AVRO_TOOLS_JAR]
                          [--inputSchemasDirectory INPUTSCHEMASDIRECTORY]
                          [--verbose]
                          version
```

My strong personal preference is to adhere to the POSIX standard dash separators, however `argparse.ArgumentParser` automatically turns each `--command-line-option` into `parser.command_line_option`, which violates the project's camel casing conventions.

The preferred solution, to me, would be to make an exception to our rules in this one case and allow `ArgumentParser` auto-generated attributes to be underscore separated. The scope of this change would be constrained to the script's `main()` method as well as whatever top-level class in the script that accepts the `argparse` generated arguments.

**Comment by diekhans** [comment]
you can specify the destination field name with dest="arvoToolsJar"

**Comment by jeromekelleher** [comment]
I tend to agree with @macieksmuga here, it would be much better to use the POSIX convention. I'm all for this, if we can do it simply. Will @diekhans idea generalise to the other CLI applications?
